8.3 rebase xs 20250328 + rebuild for updated branding-xcp-ng

stormi opened the pull request.

ydirson reviewed (commented):
Hm I'm not opposed to doing a merge of upstream branch (not sure why it was not done last time), but I'd prefer we have a separate merge to fix that before that PR.
Also do we really need to update at this point, since their rebuild does not impact us?

stormi commented:
So I split it into two merge commits. Since we're also updating branding-xcp-ng (the PR is not here yet but it's the logical follow-up to https://github.com/xcp-ng/branding-xcp-ng/pull/10), I also added a mention regarding the rebuild it will cause, so we don't rebuild xcp-ng-release just for the merge of the XS-8.3 branch.
